spec/string-functions/quote.hrx

```diff
@@ -0,0 +1,80 @@
+<===> unquoted/input.scss
+a {
+  a: quote(a);
+}
+
+<===> unquoted/output.css
+a {
+  a: "a";
+}
+
+<===>
+================================================================================
+<===> quoted-double/input.scss
+a {
+  a: quote("a");
+}
+
+<===> quoted-double/output.css
+a {
+  a: "a";
+}
+
+<===>
+================================================================================
+<===> quoted-single/input.scss
+a {
+  a: quote('a');
+}
+
+<===> quoted-single/output.css
+a {
+  a: "a";
+}
+
+<===>
+================================================================================
+<===> quote-mark-selection-single/input.scss
```

Comment:
What does "selection" indicate in this path name?

Reply:
That the `quote` function selects the most suitable outer quotes (`'` or `"`).
English is not my first language, sorry if this doesn't make sense!

Reply:
The type of quote that's printed for a string is chosen when that string is serialized to CSS; it's not an aspect of the string object itself. The `quote()` function shouldn't know anything about it.

Reply:
> The type of quote that's printed for a string is chosen when that string is serialized to CSS

Where is this specified?

`libsass` does use different types of quoting depending on the code path.

Can we keep this test here? It's conceivable that other future implementations might make the same mistake.

Reply:
We don't have a full prose specification (although we're working on building one piecemeal in [sass/sass](/sass/sass)), but there should be no observable case where the way a string is created (with single or double quotes) affects the way it's serialized in the reference implementation.

We can keep the tests (maybe name them `quote_unquoted_quote/single` and `quote_unquoted_quote/double`?), but they should have comments indicating that they're regression tests and linking to the issues in question.

Reply:
Done in https://github.com/sass/sass-spec/commit/dc5f5d551e4172eeccbb8b40922edfaf1a9777b5